### PR TITLE
Add InsanusMokrassar/TelegramBotAPI

### PR DESCRIPTION
**Pull request by madhead**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,7 +140,8 @@ Challenge your friends in MULTIPLAYER mode!
 
 #### Kotlin
 * [kotlogram](https://github.com/badoualy/kotlogram) - Easy to use and straightforward Kotlin (and Java) binding of Telegram API
-* [kotlin-telegram-bot](https://github.com/seik/kotlin-telegram-bot) - A wrapper for the Telegram Bot API written in Kotlin.
+* [kotlin-telegram-bot](https://github.com/seik/kotlin-telegram-bot) - A wrapper for the Telegram Bot API written in.
+* [TelegramBotAPI](https://github.com/InsanusMokrassar/TelegramBotAPI) - Type-safe library for work with Telegram Bot API.
```
Comment (InsanusMokrassar) [inline README.md:144]: Nowadays I am thinking to title it like `tgbotapi` :)

Comment (madhead) [reply README.md:144]: Would you change the GitHub URL?

Comment (InsanusMokrassar) [reply README.md:144]: Nope, just title : [tgbotapi]

Comment (madhead) [reply README.md]: Done!

**Pull request merged**